prinz_sample/src/main/scala/pl/touk/nussknacker/prinz/sample/SampleConfigCreator.scala

```diff
@@ -32,9 +32,14 @@ class SampleConfigCreator extends EmptyProcessConfigCreator {
     val repo = new MLFRepository()
     val response = repo.listModels
 
-    response.right.map(modelsList => modelsList.foldLeft(Map.empty[String, WithCategories[Service]])(
-      (services, model) => services + (model.getName.toString -> allCategories(new PrinzEnricher(model)))
-    )).right.get
+    val result = response.right.map(
+      modelsList => modelsList.foldLeft(Map.empty[String, WithCategories[Service]])(
+        (services, model) => services + (model.getName.name -> allCategories(new PrinzEnricher(model)))
+    ))
+    result match {
+      case Left(exception) => throw exception
+      case Right(services) => services
+    }
```

Comment:
Here we need to throw exception thrown by listModels request. However if error occurs, response.right.get will throw exception imo because it has no value.

Reply:
It's already fixed, sorry xd